pwiz_tools/Skyline/TestFunctional/AuditLogTest.cs

```diff
@@ -81,6 +81,7 @@ public void TestAuditLogLocalization()
             VerifyStringLocalization("{0:SEttings}", "{0:SEttings}");
 
             var unlocalizedMessageTypes = GetUnlocalizedMessageTypes();
+            var errMsgs = new List<string>();

```

Comment:
How can adding an empty list as the only change to this file be useful?

Reply:
yeah, dead temporary code, thanks